config/application.rb

```diff
@@ -25,6 +25,8 @@ class Application < Rails::Application
     config.load_defaults 6.0
     config.autoloader = :classic
 
+    config.action_view.sanitized_allowed_tags = ['details', 'summary', 'a']
```

Comment:
Why is this required?

Reply:
Oh, I think its not needed, I will check and remove it.

Thanks for spotting that

(Left over from attempts to fix the issue)